src/toil_scripts/lib/files.py

```diff
@@ -78,3 +135,42 @@ def consolidate_tarballs_job(job, fname_to_id):
                         tarinfo.name = os.path.join(output_name, fname, os.path.basename(tarinfo.name))
                         f_out.addfile(tarinfo, fileobj=f_in_file)
     return job.fileStore.writeGlobalFile(out_tar)
+
+
+def get_files_from_filestore(job, work_dir, input_dict):
+    """
+    Copies files from the file store to a work directory. Will expand TAR files.
+
+    :param JobFunctionWrappingJob job: passed automatically by Toil
+    :param str work_dir: current working directory
+    :param dict input_dict: {filename: fileStoreID}
+    :return: {filename: filepath}
+    :rtype: dict
+    """
+    for name, fileStoreID in input_dict.iteritems():
+        if os.path.exists(os.path.join(work_dir, name)):
+            file_path = name
+        else:
+            file_path = job.fileStore.readGlobalFile(fileStoreID, os.path.join(work_dir, name))
+        # Unpack tarfiles
+        if tarfile.is_tarfile(file_path):
+            file_path = untargz(file_path, work_dir)
+        input_dict[name] = file_path
+    return input_dict
+
+
+def untargz(input_file, output_dir):
```

Comment:
Again, not sure we need a function to solve a one-line `subprocess` call, or two lines if you're running `is_tarfile` check ahead of time.